core-bundle/assets/controllers/color-scheme-controller.js

```diff
@@ -35,27 +35,39 @@ export default class extends Controller {
     connect () {
         this.element.addEventListener('click', this.toggle);
 
-        window.matchMedia('(prefers-color-scheme: dark)').addEventListener('change', this.setLabel);
+        this.matchMedia = window.matchMedia('(prefers-color-scheme: dark)');
```

Comment:
In case you are wondering about the change: we need to remove the event listener again during disconnect (otherwise it could be added multiple times). That's why the object we're working on gets stored.